workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/GetAppModal/SelectPaymentMethod.tsx

```diff
@@ -3,56 +3,52 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import infoCircleIcon from '../../assets/icons/info_circle_icon.svg';
+import {useState} from 'react';
+
 import {Input} from '../../components/Input/Input';
 import {BillingAddress} from './BillingAddress';
 import {PaymentMethodMode} from './PaymentMethodMode';
 import {PaymentMethodSelector} from './PaymentMethodSelector';
 import {TrialTimeline} from './TrialTimeline';
 
+interface SelectPaymentMethodProps {
+	addresses: BillingAddress[];
+	billingAddress: BillingAddress;
+	email: string;
+	enableTrialMethod: boolean;
+	purchaseOrderNumber: string;
+	selectedPaymentMethod: PaymentMethodSelector;
+	setBillingAddress: (value: BillingAddress) => void;
+	setEmail: (value: string) => void;
+	setEnablePurchaseButton: (value: boolean) => void;
+	setPurchaseOrderNumber: (value: string) => void;
+	setSelectedPaymentMethod: (value: PaymentMethodSelector) => void;
+}
+
 export function SelectPaymentMethod({
 	addresses,
 	billingAddress,
 	email,
 	enableTrialMethod,
 	purchaseOrderNumber,
-	selectedAddress,
 	selectedPaymentMethod,
 	setBillingAddress,
 	setEmail,
 	setEnablePurchaseButton,
 	setPurchaseOrderNumber,
-	setSelectedAddress,
 	setSelectedPaymentMethod,
-	setShowNewAddressButton,
-	showNewAddressButton,
-}: {
-	addresses: BillingAddress[];
-	billingAddress: BillingAddress;
-	email: string;
-	enableTrialMethod: boolean;
-	purchaseOrderNumber: string;
-	selectedAddress: string;
-	selectedPaymentMethod: PaymentMethodSelector;
-	setBillingAddress: (value: BillingAddress) => void;
-	setEmail: (value: string) => void;
-	setEnablePurchaseButton: (value: boolean) => void;
-	setPurchaseOrderNumber: (value: string) => void;
-	setSelectedAddress: (value: string) => void;
-	setSelectedPaymentMethod: (value: PaymentMethodSelector) => void;
-	setShowNewAddressButton: (value: boolean) => void;
-	showNewAddressButton: boolean;
-}) {
+}: SelectPaymentMethodProps) {
+	const [selectedAddress, setSelectedAddress] = useState('');
+	const [showNewAddressButton, setShowNewAddressButton] = useState(true);
+
 	return (
-		<>
-			<div className="get-app-modal-payment-methods">
-				<div className="get-app-modal-payment-methods-container">
-					<PaymentMethodSelector
-						enableTrial={enableTrialMethod}
-						selectedPaymentMethod={selectedPaymentMethod as string}
-						setSelectedPaymentMethod={setSelectedPaymentMethod}
-					/>
-				</div>
+		<div>
```

Comment:
Why are you using this div? If it just to involve all content, you can use the <></> tag for this.

Reply:
Fixed